ui/tests/integration/components/license-test.js

```diff
@@ -26,9 +26,10 @@ module('Integration | Component | license info', function(hooks) {
   const LICENSE_WARNING_TEXT = `Warning Your temporary license expires in 30 minutes and your vault will seal. Please enter a valid license below.`;
 
   test('it renders properly for temporary license', async function(assert) {
+    const now = new Date();
     this.set('licenseId', 'temporary');
-    this.set('expirationTime', moment(moment.now()).add(30, 'minutes'));
-    this.set('startTime', moment.now());
+    this.set('expirationTime', addMinutes(now, 30));
+    this.set('startTime', now);
```

Comment:
I'm guessing this is doing some implicit conversion now (Date -> Number), so it may be better do `Date.now()` or `now.getTime()` if it should be the same as expiration time.